### ISSUE TITLE
How should standard be used next to rubocop extensions?

### ISSUE THREAD

**Issue by pablocrivella** [testdouble/standard#135]
I was wondering how should standard "integrate" with rubocop extensions like:

- https://github.com/rubocop-hq/rubocop-minitest
- https://github.com/rubocop-hq/rubocop-rspec
- https://github.com/rubocop-hq/rubocop-rails

**Comment by searls** [comment]
I haven't looked at minitest or rspec, but I recently made the call to punt on rubocop-rails (for now). Since standard is more concerned with the formatting and structure of ruby-ruby than any particular library, I'm fine with punting on library-specific gems like this.

However, I could be convinced that how Standard _ought_ to behave is to curate & choose rules to enable from each of these, but only the most important lints (prioritizing safety over style), and to only activate them only in the presence of their corresponding `rubocop-foo` gem, as opposed to making any of them a hard dep of standard's. So for instance, if a user has `rubocop-rspec` in their Gemfile, our base configuration might enable certain cops bundled in that gem and start linting them, but otherwise we'd just carry on without it.

**Comment by moger777** [comment]
Is it possible to opt into lint rules that are not part of the standard default? I know that defeats the purpose of having a standard to some extent but it is handy for say, opting into security linters that have less to do with style and more to do with preventing incorrect code.

**Comment by searls** [comment]
@moger777 no, I don't think so. I think for this case you'd need to run rubocop and standard separately, and I'm not sure that's a bad thing (first of all, rubocop is already installed; second, I believe they share a cache; third, this is how StandardJS+eslint relate to one another in similar cases)

**Comment by thijsnado** [comment]
I did the following which allows standard settings to be used in rubocop. Won't allow you to use other standard tooling but would get standard linting rules in addition to other linting rules:

```yml
require:
  - standard
  - rubocop-rails
  - rubocop-rspec

inherit_gem:
  standard: config/base.yml

Rails:
  Enabled: true
RSpec:
  Enabled: true
```

**Comment by wilg** [comment]
@thijsnado I'm noticing doing that that I don't get all of the Standard formatting – for example, it doesn't enforce double quotes if I set my `.rubocop.yml` to:

```yaml
require:
  - standard
inherit_gem:
  standard: config/base.yml
```

**Comment by thijsnado** [comment]
@wilg what version of rubocop/standard are in your Gemfile.lock?

**Comment by wilg** [comment]
@thijsnado nvm, deleted, something else was wrong, sorry!

**Comment by bf4** [comment]
see https://evilmartians.com/chronicles/rubocoping-with-legacy-bring-your-ruby-code-up-to-standard and maybe close the issue

**Comment by jmkoni** [comment]
Fixed by #236
